src/Native/ObjWriter/debugInfo/codeView/codeViewTypeBuilder.h

```diff
@@ -0,0 +1,72 @@
+//===---- codeViewTypeBuilder.h ------------------------*- C++ -*-===//
+//
+// type builder is used to convert .Net types into CodeView descriptors.
+//
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Could you please use our standard license headers on new files?

(I am fixing wrong license headers few existing files in #5544.)

Reply:
done.